src/transformers/models/big_bird/configuration_big_bird.py

```diff
@@ -91,18 +91,17 @@ class BigBirdConfig(PretrainedConfig):
     Example:
 
     ```python
+        >>> from transformers import BigBirdConfig, BigBirdModel
 
-    ```
-
-        >>> from transformers import BigBirdModel, BigBirdConfig
+        >>> # Initializing a BigBird google/bigbird-roberta-base style configuration
+        >>> configuration = BigBirdConfig()
 
-        >>> # Initializing a BigBird google/bigbird-roberta-base style configuration >>> configuration =
-        BigBirdConfig()
+        >>> # Initializing a model (with random weights) from the google/bigbird-roberta-base style configuration
+        >>> model = BigBirdModel(configuration)
 
-        >>> # Initializing a model from the google/bigbird-roberta-base style configuration >>> model =
-        BigBirdModel(configuration)
-
-        >>> # Accessing the model configuration >>> configuration = model.config
+        >>> # Accessing the model configuration
+        >>> configuration = model.config
+    ```
```

Comment:
Hi, after a close look, it seems this config docstring had some more issues previously.

Thank your for fixing this here, but let's put this line and the next line to a single line **```"""** to follow other config files :-)

Also, inside the code examples (i.e. the part with `>>>`), let's unindent those line 🙏 

Sorry for so many unexpected changes here

Reply:
And same for BigBirdPegasusConfig 🙏